google-cloud-debugger/lib/google/cloud/debugger/breakpoint/variable.rb

```diff
@@ -150,6 +169,9 @@ def initialize
           #   {Google::Cloud::Debugger::Breakpoint::Variable::MAX_DEPTH}
           # @param [Breakpoint::VariableTable] var_table A variable table
           #   to store shared compound variables. Optional.
+          # @param [Integer] limit Maximum number of bytes this convertion
```

Comment:
s/convertion/conversion/
And again two lines down. And in fact, multiple times in the examples below.